<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.1.3 (Python 3.10.0 on win32)" generated="20220111 17:28:25.287" rpa="false" schemaversion="2">
<suite id="s1" name="Yodapp" source="C:\Users\Jaque\PycharmProjects\GravidadeZERO\yodapp">
<suite id="s1-s1" name="Tests" source="C:\Users\Jaque\PycharmProjects\GravidadeZERO\yodapp\tests">
<suite id="s1-s1-s1" name="Cadastro" source="C:\Users\Jaque\PycharmProjects\GravidadeZERO\yodapp\tests\cadastro.robot">
<test id="s1-s1-s1-t1" name="Email incorreto">
<kw name="Start Session" library="base" type="SETUP">
<kw name="New Browser">
<arg>chromium</arg>
<arg>headless=False</arg>
<msg timestamp="20220111 17:28:25.422" level="FAIL">No keyword with name 'New Browser' found.</msg>
<status status="FAIL" starttime="20220111 17:28:25.421" endtime="20220111 17:28:25.422"/>
</kw>
<kw name="New Page">
<arg>${BASE_URL}</arg>
<status status="NOT RUN" starttime="20220111 17:28:25.423" endtime="20220111 17:28:25.423"/>
</kw>
<status status="FAIL" starttime="20220111 17:28:25.420" endtime="20220111 17:28:25.423"/>
</kw>
<kw name="End Session" library="base" type="TEARDOWN">
<kw name="Take Screenshot">
<msg timestamp="20220111 17:28:25.425" level="FAIL">No keyword with name 'Take Screenshot' found.</msg>
<status status="FAIL" starttime="20220111 17:28:25.425" endtime="20220111 17:28:25.425"/>
</kw>
<status status="FAIL" starttime="20220111 17:28:25.424" endtime="20220111 17:28:25.425">No keyword with name 'Take Screenshot' found.</status>
</kw>
<tag>email_inv</tag>
<status status="FAIL" starttime="20220111 17:28:25.420" endtime="20220111 17:28:25.426">Setup failed:
No keyword with name 'New Browser' found.

Also teardown failed:
No keyword with name 'Take Screenshot' found.</status>
</test>
<doc>Suite de testes do cadastro de personagens</doc>
<status status="FAIL" starttime="20220111 17:28:25.308" endtime="20220111 17:28:25.427"/>
</suite>
<status status="FAIL" starttime="20220111 17:28:25.306" endtime="20220111 17:28:25.429"/>
</suite>
<status status="FAIL" starttime="20220111 17:28:25.288" endtime="20220111 17:28:25.429"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
<stat pass="0" fail="1" skip="0">email_inv</stat>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Yodapp">Yodapp</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1" name="Tests">Yodapp.Tests</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1-s1" name="Cadastro">Yodapp.Tests.Cadastro</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20220111 17:28:25.412" level="ERROR">Error in file 'C:\Users\Jaque\PycharmProjects\GravidadeZERO\yodapp\resources\base.robot' on line 4: Initializing library 'Browser' with no arguments failed: Couldn't execute node. Please ensure you have node.js installed and in PATH. See https://nodejs.org/ for instructions. Original error is [WinError 2] O sistema n√£o pode encontrar o arquivo especificado
Traceback (most recent call last):
  File "C:\Users\jaque\PycharmProjects\GravidadeZERO\venv\lib\site-packages\Browser\browser.py", line 718, in __init__
    self.playwright = Playwright(
  File "C:\Users\jaque\PycharmProjects\GravidadeZERO\venv\lib\site-packages\Browser\playwright.py", line 50, in __init__
    self.ensure_node_dependencies()
  File "C:\Users\jaque\PycharmProjects\GravidadeZERO\venv\lib\site-packages\Browser\playwright.py", line 65, in ensure_node_dependencies
    raise RuntimeError(</msg>
</errors>
</robot>
